### PR TITLE
removing the segmentation fault

### PR DESCRIPTION
**Pull request by Jitendra-Rajak**
#### Describe Bug or Feature
Server segmentation fault while trying to access a void pointer value.

#### Describe Your Change
Added a check before accessing the void pointer value, if the void pointer hold any value than only try to access. 

#### Attach Test and Valgrind Logs/Output
Logs attached to the ticket.

### CODE REVIEW

File: src/server/licensing_func.c [hunk 1]
```diff
@@ -544,6 +544,9 @@ license_one_node(pbsnode *pnode)
 				update_license_highuse();
 			} else {
 				add_to_unlicensed_node_list(pnode);
+				if (pnode->nd_attr[ND_ATR_LicenseInfo].at_val.at_long) {
+					licensing_control.licenses_total_needed += pnode->nd_attr[ND_ATR_LicenseInfo].at_val.at_long;
```
Comment (arungrover) [inline src/server/licensing_func.c:548]: Please use get_attr_l instead of accessing it directly.

File: src/server/licensing_func.c [hunk 2]
```diff
@@ -694,8 +697,10 @@ init_licensing(struct work_task *ptask)
 		if (pbsndlist[i]->nd_attr[ND_ATR_LicenseInfo].at_val.at_long) {
 			licensing_control.licenses_total_needed += pbsndlist[i]->nd_attr[ND_ATR_LicenseInfo].at_val.at_long;
 		} else {
-			set_node_lic_info_attr(pbsndlist[i]);
-			licensing_control.licenses_total_needed += pbsndlist[i]->nd_attr[ND_ATR_LicenseInfo].at_val.at_long;
+			if (pbsndlist[i]->nd_lic_info != NULL) {
+				set_node_lic_info_attr(pbsndlist[i]);
+				licensing_control.licenses_total_needed += pbsndlist[i]->nd_attr[ND_ATR_LicenseInfo].at_val.at_long;
```
Comment (arungrover) [inline src/server/licensing_func.c:702]: Please use get_attr_l instead of accessing it directly.

**Review by arungrover — CHANGES_REQUESTED** [review]

File: src/server/licensing_func.c (hunk 1 above)
Comment (Jitendra-Rajak) [reply src/server/licensing_func.c]: made the changes please have a look @arungrover

File: src/server/licensing_func.c (hunk 2 above)
Comment (Jitendra-Rajak) [reply src/server/licensing_func.c]: done

**Review by arungrover — APPROVED** [review]

**Review by subhasisb — APPROVED** [review]

**Pull request merged**